java-checks-test-sources/src/main/java/checks/TestClassAndMethodVisibilityCheckTest.java

```diff
@@ -0,0 +1,22 @@
+package checks;
+
+import org.junit.jupiter.api.Test;
+
+class TestClassAndMethodVisibilityCheckTest {
+
+  @Test
+  public void testPublic() {} // Noncompliant [[sc=3;ec=9]] {{Remove this access modifier}}
```

Comment:
Usually, when the message is constant, we test it only once.
So, here it's ok, you can use `// Noncompliant [[sc=3;ec=9]] {{Remove this access modifier}}`
And elsewhere it's enough to use `// Noncompliant`, without the location and the message